features/api/adherents.feature

```diff
@@ -0,0 +1,29 @@
+Feature:
+  In order to get adherents' information
+  As a referent
+  I should be able to acces adherents API data
+
+  Background:
+    Given the following fixtures are loaded:
+      | LoadUserData      |
+      | LoadAdherentData  |
+
+  Scenario: As a non logged-in user I can not access the adherents count information
+    When I am on "/api/adherents/count"
+    Then the response status code should be 200
```

Comment:
In another PR, we should change this behavior to answer with 401 on /api route (like we did on data-api)